django_react_proj/users-fe/src/components/RealTime/RealTime.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import moment from 'moment'
+import styles from "./RealTime.module.css"
+
+const useCurrentCallback = (callback) => {
+    const reference = React.useRef();
+    reference.current = callback;
+    return (...args) => {
+      return reference.current?.(...args);
+    };
+  };
+  
+const Time = () => {
+    const [time, setTime] = React.useState(0);
+    const currentCallback = useCurrentCallback(() => {
+    const date = new Date();
+    setTime(date.toISOString());
+    });
+    React.useEffect(() => {
+    const handle = setInterval(currentCallback, 100);
```

Comment:
If you want to show time up to seconds 100ms scrap time is too much. Use a full second, which is 1000ms

Reply:
Changed